sources/packages/backend/apps/workers/src/controllers/disbursement/_tests_/e2e/disbursement.controller.saveDisbursementSchedules.e2e-spec.ts

```diff
@@ -344,6 +344,119 @@ describe("DisbursementController(e2e)-saveDisbursementSchedules", () => {
     assertOveraward(overawards, "CSLP", 300);
   });
 
+  it("Should assert disbursements do not have estimated awards and disbursement value amounts are zero.", async () => {
```

Comment:
`"Should assert disbursements must have 'hasEstimatedAward' flag as false when disbursement value amounts are zero."` ?